taskboot/cli.py

```diff
@@ -251,7 +277,7 @@ def main():
         help="Github repository name to use (example: mozilla/task-boot)",
     )
     github_release_cmd.add_argument(
-        "version", type=str, help="Release version tag to create or update on github"
+        "version", type=str, help="Release version tag to create or update on github",
```

Comment:
I imagine the formatting changes were applied by black and not you, right?

Reply:
Exactly